packages/office-ui-fabric-react/src/components/Slider/Slider.types.ts

```diff
@@ -117,10 +117,12 @@ export interface ISliderStyleProps {
   rootIsHorizontal?: boolean;
   rootIsVertical?: boolean;
   showTransitions?: boolean;
+  showValue?: boolean;
 }
```

Comment:
I know this was already here but we've been doing something like this to more tightly bind it to the component's props interface:

```
export type ITeachingBubbleStyleProps =
  Required<Pick<ITeachingBubbleProps, 'theme'>> &
  Pick<ITeachingBubbleProps, 'hasCondensedHeadline' | 'hasSmallHeadline' | 'isWide'> & {
    /** Class name for callout. */
    'calloutClassName'?: string
  };
```

Reply:
done